### ISSUE TITLE
Draw the boundary of the agroforestry system

### ISSUE THREAD

**Issue by mmaelicke** [hydrocode-de/integra-frontend#129]

**Comment by JesJehle** [comment]
Not that big of a deal, but if the hover only works some of the time.

**Comment by mmaelicke** [comment]
Yeah, I am aware of that. The mouseleave is not fired correctly, when you move the mouse from feature to feature. If there is some space between the features, it works fine.
The issue is known in mapbox, there is also a workaround using mousemove. I will fix that at some point
